### DISCUSSION TITLE
Which attribute shall I use to know if the heater is on ?

### DISCUSSION THREAD

**Discussion by Merangle** [jmcollin78/versatile_thermostat#207] [General]
First comment: 
It seems that there are more attributes available in the developper tools / states that in the documentation. 
Do you think it could be possible to update the documentation ? 

I was looking for an attribute which will show when the heater is on. 
is there an attribute available ? 

Thanks for your help.

**Comment by jmcollin78** [comment]
`is_device_active` est l'attribut que tu cherches. Oui ok pour la Doc. Ça a bcp bougé ces derniers temps.

**Comment by Merangle** [comment]
C'est is_device_active
Merci

Le sam. 18 nov. 2023 à 21:01, Jean-Marc Collin ***@***.***> a
écrit :
